taqueria-plugin-taquito/transfer.ts

```diff
@@ -122,7 +115,20 @@ const performTransferOp = (tezos: TezosToolkit, contractInfo: TableRow): Promise
 			},
 		})
 		.then(op => op.confirmation().then(() => op.hash))
-		.catch(err => sendAsyncErr(`Error during transfer operation:\n${err} ${JSON.stringify(err, null, 2)}`));
+		.catch(err => {
+			if (err instanceof Error) {
+				if (/empty_implicit_contract/.test(err.message)) {
+					const result = (err.message).match(/(?<="implicit":")tz[^"]+(?=")/);
+					const publicKeyHash = result ? result[0] : undefined;
+					if (publicKeyHash) {
+						return sendAsyncErr(
+							`${publicKeyHash} possibly has no funds. Go to https://teztnets.xyz and click "Faucet" of the desired protocol to fund it. Note that you might need to wait for a few seconds for the network to register the funds.`,
```

Comment:
The root account {publicKeyHask} for the target environment may not be funded \n
To fund this account:
1. Go to https://teztnets.xyz and click "Faucet" of the target testnet
2. Copy and paste {publicKeyHash} into the 'wallet address field
3. Request some Tez
(Note that you might need to wait for a few seconds for the network to register the funds)